azure-toolkit-libs/pom.xml

```diff
@@ -80,7 +80,7 @@
         <json.schema.validator.version>2.2.14</json.schema.validator.version>
         <groovy.version>2.5.11</groovy.version>
         <junit.version>4.13.1</junit.version>
-        <lombok.version>1.18.16</lombok.version>
+        <lombok.version>1.18.18</lombok.version>
```

Comment:
why? need to prevent version conflicts with other modules and intellij plugin

Reply:
intellij lombok plugin needs this change, otherwise the lombok java will not compile